src/Components/ServiceScene/Breakdowns/Patterns/PatternFieldLabelStatsRow.tsx

```diff
@@ -0,0 +1,82 @@
+import { css, cx } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  logsStatsRow: css`
+    label: logs-stats-row;
+    margin: ${parseInt(theme.spacing(2), 10) / 1.75}px 0;
```

Comment:
theme.spacing(1) is 4. This could give you a similar number, but I would recommend to always use whole numbers or quarters for spacing. E.g. spacing(1), spacing(0.25), spacing(3.5).


```suggestion
    margin: ${theme.spacing(1.15)}px 0;
```

Reply:
This is done.